ocp-templates/app-interface/backend.yaml

```diff
@@ -91,6 +91,11 @@ objects:
               secretKeyRef:
                 key: name
                 name: ibutsu-superadmin
+          - name: JWT_SECRET
```

Comment:
@bsquizz what do we need to do on the AppSRE side? Add it to vault?

Reply:
Yep, we will add it to vault and then update the app-interface config for our namespace to insert the secret into the ns.